### DISCUSSION TITLE
HMI_AU_MIB2_S_EU_G21

### DISCUSSION THREAD

**Discussion by mikemihgk** [Mr-MIBonk/M.I.B._More-Incredible-Bash#98] [Q&A]
Hello I am new here.
Don’t know lot things but I want to ask if I can activate some extra for my car .
I want to enable , CarPlay, navigation and anything else is good .
My car is Audi q2 2021 ( December) model
My unit wrote that IN RED LETTERS

HMI_AU_MIB2_S_EU_G21_Q2PA_ (numbers)

In another screen show that 
MST2_EU_AU_PO(numbers)

MU:153

Framework MIB2MAIN

any help is acceptable.
 I am new here so I don’t know lot. I read all article but I need to be sure if work for my car
 So pls help me if y can

**Comment by harman-f** [comment]
MHI2 systems here only :)
